### PR TITLE
Refactoring Facilities for iziModal

### PR DESCRIPTION
**Pull request by apooravc**
Added iziModal to Add Facility and Edit Facility. Some issues still coming up:
1. Edit Facility modal opens upon clicking on a facility link repeatedly. Issue associated with fetching of dynamic links.
2. Edit Facility modal isn't getting closed after clicking Cancel.

One way to see the changes _locally_ would be as follows:
1) Clone my repo via `git clone https://github.com/apooravc/LibreEHR.git`
2) After cloning, only the master branch would be available locally. Enter `git checkout -b iziModal-facilities origin/iziModal-facilities` to get the branch _iziModal-facilities_ locally.
3) The changes could then be seen on facilities page.

@teryhill @muarachmann Please have a look.

### CODE REVIEW

**Comment by muarachmann** [comment]
Your code looks ok put the izi frame out of the tr tag rather put if just after the body since we all know it will be hidden or so by the css. I dont know y but i guess it searched the doc for that div and it takes time to find it

**Comment by muarachmann** [comment]
We clearified this on the chat forum @teryhill works fine now ;) 
@apooravc dont forget to get the hospital names dynamically like so
`var editTitle = $(this).children('span').text()` and parse it via izi

**Comment by muarachmann** [comment]
kazam_screenshot_00012

should look like so 
you can do for the sub title Address, city and telephone number thats upto you

**Comment by apooravc** [comment]
Fixed remaining issues, thanks to @muarachmann

File: interface/usergroup/facilities.php [hunk 1]
```diff
@@ -92,48 +92,92 @@
...
-        'frameWidth' : 650        
+    $("#addFacilities-iframe").iziModal({
+      title: 'Add Facility',
+      subtitle: 'Enter details about the new facility',
+      headerColor: '#88A0B9',
+      closeOnEscape: true,
+      fullscreen: true,
+      overlayClose: false,
+      closeButton: true,
+      theme: 'light',
```
Comment (aethelwulffe) [inline interface/usergroup/facilities.php]: No other way of calling this stuff?  Is the "light" theme an LEHR light theme, or a component theme?

File: interface/usergroup/facilities.php [hunk 2]
```diff
@@ -92,48 +92,92 @@
...
+        closeOnEscape: true,
+        fullscreen: true,
+        overlayClose: false,
+        closeButton: true,
+        theme: 'light',
+        iframe: true,
+        width: 700,
+        focusInput: true,
+        padding: 5,
+        iframeHeight: 350,
```
Comment (aethelwulffe) [inline interface/usergroup/facilities.php]: Going to test moving and resizing (or if it is needed).

File: interface/usergroup/facilities_add.php [hunk 3]
```diff
@@ -130,33 +115,13 @@ function toggle( target, div ) {
                     toggle( $(this), "#DEM" );
                 });
 
-                // fancy box
-                enable_modals();
-
                 tabbify();
-
-                // special size for
-                $(".large_modal").fancybox( {
```
Comment (aethelwulffe) [inline interface/usergroup/facilities_add.php]: So glad to see this stuff go bye-bye.

File: interface/usergroup/facilities_add.php [hunk 4]
```diff
@@ -44,28 +44,13 @@
 <html>
     <head>
         <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-        <link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['webroot'] ?>/library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
         <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
         <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.1.3.2.js"></script>
```
Comment (aethelwulffe) [inline interface/usergroup/facilities_add.php]: Some of the library calls here need updating, kike jquery-ui.js.  anchorposition and popupwindow should be included in the header file as options as well.

File: interface/usergroup/facility_admin.php [hunk 5]
```diff
@@ -12,28 +12,14 @@
...
 <html>
 <head>
 
   <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-  <link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
   <script type="text/javascript" src="../../library/dialog.js"></script>
   <script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
   <script type="text/javascript" src="../../library/js/common.js"></script>
-  <script type="text/javascript" src="../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
   <script src="<?php echo $GLOBALS['standard_js_path']; ?>anchorposition/AnchorPosition.js"></script>
```
Comment (aethelwulffe) [inline interface/usergroup/facility_admin.php]: Headers/includes could use some cleanup.

**Comment by aethelwulffe** [comment]
Dudes.  If you review and test another's code.  Do a review and approve here.

File: interface/usergroup/facilities.php (hunk 1 above)
Comment (apooravc) [reply interface/usergroup/facilities.php]: It's a theme of iziModal itself.

**Comment by aethelwulffe** [comment]
@apooravc Please resolve the facilities.php conflicts here (does not look hard) and I will get this merged.

**Comment by apooravc** [comment]
@aethelwulffe Resolved merge conflicts. But regarding that iziModal conflict with updater, the fix would also require changing `parent.$(".fa-refresh").click()` in iziModal code (in initIziLink(link) function) to `parent.$(".tab-refresh-icon").click()`. The custom `tab-refresh-icon` class is added to all refresh icon element of tabs in #1194. Should I add it here?

**Comment by teryhill** [comment]
@apooravc keep them separate for now. So this one is ready to merge.

**Comment by aethelwulffe** [comment]
Yes, separate is fine.
-As soon as we get the second review, this one should go first, being the oldest etc... and about 34 days old.

**Comment by aethelwulffe** [comment]
Just did a fresh test on top of the other facility stuff.  Perfect.  Not that the /usergroup files would have any specific interaction.

**Comment by aethelwulffe** [comment]
-There was some context hovertext in the UI before that seems missing now,

**Comment by teryhill** [comment]
Code looks good to me I have not tested but @aethelwulffe has

**Pull request merged**